### ISSUE TITLE
Can't create a project with nx and Angular 17

### ISSUE THREAD

**Issue by geromegrignon** [analogjs/analog#762]
### Please provide the environment you discovered this bug in.

Environment: terminal used to create/add a analog project with nx

### Which area/package is the issue in?

create-analog

### Description

By either starting a new nx workspace or adding a analog project in an existing project, the following error appears when nx tries to generate the project.

However the path in the error message sounds right based on the repository structure.

### Please provide the exception or error you saw

```
NX   generateFiles: No files found in "/Users/gerome/Documents/workspace/oneshots/angular-space/node_modules/@analogjs/platform/src/lib/nx-plugin/src/generators/app/files/template-angular-v17".
Are you sure you specified the correct path?
```

### Other information

So far i can't find the reason but wold be happy to fix it with guidance.

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [ ] No

**Comment by geromegrignon** [comment]
Is it due to `analog/platform` nx-plugin v17 support changes 3 days ago are not published on npm yet?

**Comment by draylegend** [comment]
Tested today, no errors. Each step was successful using `npm create analog` and `npm init analog`

**Comment by geromegrignon** [comment]
> Tested today, no errors. Each step was successful using `npm create analog` and `npm init analog`

The issue is about creating an analog project with nx, as explained on this page: https://analogjs.org/docs/integrations/nx

**Comment by draylegend** [comment]
Missing folder `template-angular-v17`
![image](https://github.com/analogjs/analog/assets/16114644/f752967d-84d2-43d3-b8df-e30b3ece12d5)
> https://www.npmjs.com/package/@analogjs/platform?activeTab=code

**Comment by brandonroberts** [comment]
This has been resolved with the latest release of Analog (0.2.21)
